### PR TITLE
Add Image Query Task docs

### PR DESCRIPTION
**Pull request by andrewfrench**
Adds missing documentation for the Image Query Task.

### CODE REVIEW

File: docs/griptape-framework/structures/tasks.md [hunk 1]
```diff
@@ -596,3 +596,38 @@ task = OutpaintingImageGenerationTask(
...
+
+```python
+from griptape.engines import ImageQueryEngine
+from griptape.drivers import OpenAiVisionImageQueryDriver
+from griptape.tasks import ImageQueryTask
+
+
+# Create a driver configured to use OpenAI's GPT-4 Vision model.
+driver = OpenAiVisionImageQueryDriver(
+    max_tokens=100,
```
Comment (emjay07) [inline docs/griptape-framework/structures/tasks.md:614]: I remember that this `max_tokens` was pretty important for this functionality. Is that specific to the Vision Driver or all Drivers? Is that something we want to call out here (or in the Driver docs?)

Comment (andrewfrench) [reply docs/griptape-framework/structures/tasks.md:614]: That is a specific quirk of the Vision driver and it's mentioned on its page.

File: docs/griptape-framework/structures/tasks.md [hunk 2]
```diff
@@ -596,3 +596,38 @@ task = OutpaintingImageGenerationTask(
...
+
+# Load the input image artifact.
+with open("tests/assets/mountain.png", "rb") as f:
+    image_artifact = ImageLoadaer().load(f.read())
+
+# Create a task configured to use the engine.
+task = ImageQueryTask(
+    input=("Describe the weather in this image", [image_artifact]),
+    image_query_engine=engine,
+)
```
Comment (collindutter) [inline docs/griptape-framework/structures/tasks.md:630]: Do we use tasks directly like this elsewhere in the docs? Seems like a Task should always be paired with a Structure.

Comment (andrewfrench) [reply docs/griptape-framework/structures/tasks.md:630]: Yes, directly above, but those are also my additions. I suppose this reflects the way we provide direct examples of engines, but if this truly doesn't make sense for a task I can update.

**Review by collindutter — APPROVED** [review]

**Review by emjay07 — APPROVED** [review]

**Pull request merged**